Makefile

```diff
@@ -33,6 +34,16 @@ test:
 		go test $(TEST_BUILDFLAGS) ./pkg/... ./cmd/... -coverprofile cover.out; \
 	fi
 
+.PHONY: pact-consumer
+pact-consumer:
+	mkdir -p pacts/consumer && ( rm -rf pacts/consumer/* || : )
+	go test $(TEST_BUILDFLAGS) -v ./contracts/... || true
```

Comment:
Adding these "|| true" for safety since this cannot be tested until next release.